src/base/bittorrent/session.cpp

```diff
@@ -2314,10 +2375,19 @@ bool Session::loadMetadata(const MagnetUri &magnetUri)
     p.save_path = Utils::Fs::toNativePath(savePath).toStdString();
 
     // Forced start
+#if LIBTORRENT_VERSION_NUM < 10200
     p.flags &= ~libt::add_torrent_params::flag_paused;
     p.flags &= ~libt::add_torrent_params::flag_auto_managed;
+#else
+    p.flags &= ~libt::torrent_flags::paused;
+    p.flags &= ~libt::torrent_flags::auto_managed;
+#endif
     // Solution to avoid accidental file writes
+#if LIBTORRENT_VERSION_NUM < 10200
     p.flags |= libt::add_torrent_params::flag_upload_mode;
+#else
+    p.flags &= ~libt::torrent_flags::upload_mode;
```

Comment:
This doesn't look right. in the first version you ADD the upload mode flag, in the bottom one you REMOVE the upload mode flag